src/setup_payload/QRCodeSetupPayloadParser.h

```diff
@@ -43,14 +44,26 @@ class QRCodeSetupPayloadParser
     std::string mBase38Representation;
 
 public:
+    /**
+     * base38Representation is expected to start with the "MT:" prefix, in general.
+     */
     QRCodeSetupPayloadParser(std::string base38Representation) : mBase38Representation(std::move(base38Representation)) {}
     CHIP_ERROR populatePayload(SetupPayload & outPayload);
```

Comment:
Please document what this method does if the string contains multiple payloads.

Reply:
Will do.  It errors out.

Reply:
Done.